package/models.py

```diff
@@ -375,6 +380,32 @@ def fetch_pypi_data(self):
                 if self.pypi_downloads is None:
                     self.pypi_downloads = total_downloads
 
+                # get documents_url from pypi
+                if not self.documentation_url:
+                    if docs_url := info["project_urls"].get("Documentation"):
+                        self.documentation_url = docs_url
+
+                    elif docs_url := info["project_urls"].get("Docs"):
+                        self.documentation_url = docs_url
+
+                    elif docs_url := info["project_urls"].get("docs"):
+                        self.documentation_url = docs_url
+
+                    elif docs_url := info["project_urls"].get("documentation"):
+                        self.documentation_url = docs_url
+
+                    elif docs_url := info.get("docs_url"):
+                        self.documentation_url = docs_url
+                    elif (
+                        info.get("description")
+                        and info.get("description_content_type") == "text/markdown"
+                    ):
+                        self.documentation_url = (
+                            extract_documentation_url_from_markdown(
+                                info.get("description")
+                            )
+                        )
```

Comment:
I think we can limit the scope to only what is structured data from PyPI JSON API. 

We have another tool we worked on which maintainers can use to help them score better that we still need to hook up.

Reply:
I agree.

Reply:
removed